TaskVlopper/Models/WorklogViewModels.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaskVlopper.Base.Model;
+
+namespace TaskVlopper.Models
+{
+    public class WorklogsViewModel
```

Comment:
Same here with plural class names. Especially when not corresponding with filename